iree/tools/utils/cpu_features.c

```diff
@@ -0,0 +1,165 @@
+// Copyright 2022 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree/tools/utils/cpu_features.h"
+
+#include "iree/base/target_platform.h"
+
+#if defined IREE_PLATFORM_LINUX && defined IREE_ARCH_ARM_64
+#include <asm/hwcap.h>
+#include <sys/auxv.h>
+#endif
+
+#if defined IREE_ARCH_ARM_64
```

Comment:
use parens to avoid potential tokenization issues (and match style):
```suggestion
#if defined(IREE_ARCH_ARM_64)
```
(here and elsewhere)

Reply:
ok  (I dont understand the potential tokenization issue here, but fine)